java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/model/PullRequestStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.google.cloud.model;
+
+public record PullRequestStatus(long created, long merged, Interval interval) {
```

Comment:
Can you add Javadoc, especially which part of the dependency report this helps, with an example value.

Reply:
We can change the class name to `PullRequestStatistics`.

Reply:
Done.